### PR TITLE
Buttons and badges

### PR DESCRIPTION
**Pull request by rahulyadav139**
Button and badge components are added.

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -7,25 +7,28 @@
...
+.avatar-with-image.medium,
+.avatar-with-text.medium {
   width: 100px;
   height: 100px;
 }
-.avatar-large {
+.avatar-with-image.large,
+.avatar-with-text.large {
   width: 150px;
   height: 150px;
```
Comment (tarunsankhla) [inline components/avatar/avatar.css:33]: I think rem should be preferable here

**Review by tarunsankhla — COMMENTED** [review]
Looks great only one thing try using rem or em and avoid giving px for bigger scale

File: components/avatar/avatar.css (hunk 1 above)
Comment (rahulyadav139) [reply components/avatar/avatar.css:33]: I forgot to change to rem in the final version. Thank you for pointing that out.

**Comment by muskanbagrecha** [comment]
Looks great. Well documented. Well thought of variants for buttons.

**Comment by muskanbagrecha** [comment]
1. In "button with icon and text", some space can be given after text. 
2. Hover effect is very subtle on links (primary link). Almost not visible. You can enhance it if you want. 
3. Overall, looks great.

**Review by m26unkwn — APPROVED** [review]
You can give some line-height to the paragraph. otherwise, everything is clean.

**Comment by rahulyadav139** [comment]
Thank you so much for the review. I have updated the file accordingly.

**Pull request merged**